types/plupload/tsconfig.json

```diff
@@ -2,9 +2,10 @@
     "compilerOptions": {
         "module": "commonjs",
         "lib": [
-            "es6"
+            "es6",
+            "dom"
         ],
-        "noImplicitAny": true,
+        "noImplicitAny": false,
```

Comment:
Any specific reason you had to change this... in general shouldn't be required when making changes to an existing type definition.

Reply:
Alright, back at the keyboard.  So yeah, this:

```
static Uploader(settings: plupload_settings): void;
```

Always resulted in an `any` type in both my implementation and in the tests.  I couldn't find a way to define it without furthering the scope of what I needed to change at the time.  To fix this noImplicityAny change back, we'd need to expand on the typings of `Uploader`.  I believe.  It's been a minute since I've worked on a typing, but I've never done this method of defining a constructor.

Reply:
I took another look.. I think what you want is something like:
```typescript
declare namespace plupload {

    class Uploader{
        constructor(settings:any);
    }
    /**
     * Executes the callback function for each item in array/object. If you return false in the
     * callback it will break the loop.
     *
     * @method each
     * @static
     * @param {Object} obj Object to iterate.
     * @param {function} callback Callback function to execute for each item.
     */
    function each(obj: any, callback: Function): void;
}
```

Reply:
Sounds good to me, and I'll gladly make the change.  I'm just not sure if that breaks anything for plupload users of lower 2.x versions.  @patrickbussmann you know any more on this?

Reply:
Well... That does make the static methods hanging off of `plupload` a pain.  Since you can't instantiate plupload, just the Uploader.

Reply:
Just went through and followed your advice, and the only thing I'm not sure how to handle now are the static constant declarations like `VERSION`, `STOPPED`, etc.

Reply:
and the random interfaces that define that they want pluploader, while it seems like the intent there is the pluploader.Uploader class, I'm not entirely sure about it.  This is why I didn't mess around with it.  This is also why we should just write everything in typescript to begin with.  😆